molecule/common/verify.yml

```diff
@@ -25,6 +25,7 @@
           tasks_from: test_qdr
         vars:
           qdr_container_name: 'qdrtest'
+          container_bin: docker
```

Comment:
Thanks! I had just spotted this on the functional test side last night. Yay!

Reply:
Here lies one of the benefits of collaborating across timezones